assets/monitoring-pgt-overlay.yaml

```diff
@@ -0,0 +1,18 @@
+applications:
+  prometheus:
+    charm: cs:prometheus2
+    constraints: "mem=4G root-disk=16G"
+    num_units: 1
+  grafana:
+    charm: cs:grafana
```

Comment:
We don't have stable charms yet (pending IS promotion), but we do have them in beta/edge if you'd like to lock these to a revision:
```
cs:grafana-30
cs:prometheus2-11
```
I dunno if it's worth the headache of having to keep this asset up2date as revs change, but it would give users a fighting chance to have this document actually work.

Reply:
both revs are now in stable; no need to pin here.